ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/PhotoEdit.java

```diff
@@ -13,55 +13,50 @@
 public class PhotoEdit {
```

Comment:
The heavy usage of synchronized might indicate that Photo should be immutable and each modification step should produce a new instance which is held in an atomic reference.